src/common/meta/src/kv_backend/txn.rs

```diff
@@ -121,6 +129,21 @@ pub struct Txn {
 }
 
 impl Txn {
+    pub fn merge_all<T: IntoIterator<Item = Txn>>(values: T) -> Self {
+        // Safety: values must contain an element.
+        values.into_iter().reduce(|acc, e| acc.merge(e)).unwrap()
```

Comment:
Can we use `unwrap_or_default()` ? It seems that it is not possible to restrict the iterator to contain at least one element.